testing/helpers/l10n/cldrCalendarDataFa.js

```diff
@@ -0,0 +1,543 @@
+const Globalize = require('globalize');
+
+Globalize.load({
+    'main': {
```

Comment:
Where did we get this data?

Reply:
From "cldr-dates-full" package. It's "ca-gregorian.json" for "fa" locale

Reply:
@ksercs @nightskylark 
Wouldn't it be better to add this package to "devDependencies" and load these files from node_modules?

Reply:
https://unpkg.com/browse/devextreme-cldr-data@1.0.2/fa.js